### PR TITLE
Membership form switch

### PR DESCRIPTION
**Pull request by KaiNissen**

### CODE REVIEW

File: app/templates/MembershipApplication.html.twig [hunk 1]
```diff
@@ -20,16 +20,21 @@
...
-
-	// TODO add initial validation status to show validation errors when coming from the server
+<script id="init-form"
+		data-initial-form-values='{% if initialFormValues %}{$ initialFormValues|json_encode|e("html_attr") $}{% else %}{}{% endif %}'
+		data-violated-fields='{% if violatedFields %}{$ violatedFields|json_encode|e("html_attr") $}{% else %}{}{% endif %}'
+		data-validate-amount-url='{$ basepath|e("html_attr") $}/validate-amount'
+		data-validate-address-url='{$ basepath|e("html_attr") $}/validate-address'
+		data-validate-iban-url='{$ basepath $}/check-iban'
+		data-generate-iban-url='{$ basepath $}/generate-iban'
+	>
```
Comment (gbirke) [inline app/templates/MembershipApplication.html.twig]: Ha, now it's your turn to make the quotes consistent ;-)

**Comment by gbirke** [comment]
Switching back and forth works, but there is no validation before going to the SEPA page. Will you do this in another commit?

**Comment by KaiNissen** [comment]
Yes, validation is up next.

File: app/templates/MembershipApplication.html.twig (hunk 1 above)
Comment (KaiNissen) [inline app/templates/MembershipApplication.html.twig]: Wow, did I just copy and paste it?!

**Comment by gbirke** [comment]
Don't worry, #398 fixes this build too.

**Comment by KaiNissen** [comment]
I already stopped worrying, when I noticed the build failing was totally unrelated. :) Thanks for fixing it!

**Pull request merged**